docs/configure.rst

```diff
@@ -0,0 +1,111 @@
+.. highlight:: console
+
+
+.. _configure:
+
+=========
+Configure
+=========
+
+This page shows you how to install `Croud`_, the dedicated command-line
+interface (CLI) and log in to CrateDB Cloud. Croud lets you interact with
+CrateDB Cloud — from creating organizations and projects to deploying clusters.
+
+.. CAUTION::
+
+   Croud requires `Python 3.6 or higher`_.
+
+.. NOTE::
+
+  This section assumes that you have `created a CrateDB Cloud account`_.
+
+Install Croud
+=============
+
+Install Croud by running the following command:
+
+.. code-block:: console
+
+    sh$ python3 -m pip install --user -U croud
+
+Next, verify if the installation was successful by running:
+
+.. code-block:: console
+
+    sh$ croud -v
+
+If Croud is installed, the command returns the Croud version
+number, like so:
+
+.. code-block:: console
+
+   croud 0.16.0.
+
+.. NOTE::
+
+  Make sure to always `update Croud to the latest version`_ to ensure it has all the latest features.
+
+.. _sign-in:
+
+Log in to CrateDB Cloud
+=======================
+
+To log in to CrateDB Cloud, run the following command:
+
+.. code-block:: console
+
+    sh$ croud login
+
+The CrateDB Cloud sign-in page opens up:
+
+.. image:: _assets/img/cloud-sign-in-page.png
+
+Enter your username and password and select *Sign in*.
+
+If successful, you should get the following message:
+
+`You have successfully logged into CrateDB Cloud!`
+
+Verify that you are logged in to CrateDB Cloud by running this command:
+
+.. code-block:: console
+
+    sh$ croud me
+
+If you're logged in, you should get your email address, and username returned, like so:
+
+.. code-block:: console
+
+   +--------------------+--------------+
+   | email              | username     |
+   |--------------------+--------------|
+   | youruser@email.com | yourusername |
+   +--------------------+--------------+
+
+If you're not logged in, give the following command again:
+
+.. code-block:: console
+
+  sh$ croud login
+
+Once you're logged in, use the following link to open up the login page and access the Cloud Console:
+
+westeurope.azure.cratedb.cloud
+
+Once you're logged in, you should see the Cloud Console.
+
+.. image:: _assets/img/cloud-console-after-sign-up.png
+
```

Comment:
needs some further explanation here. why is the screen so empty etc, etc. hold the reader's hand